release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1151,7 +1188,12 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 [id="ocp-4-10-samples-bug-fixes"]
 ==== Samples
 
-* Before this update, if the Cluster Samples Operator encountered an `APIServerConflictError` error, it reported `sample-operator` as having `Degraded` status until it recovered. Momentary errors of this type are not unusual during upgrades and were causing undue concern for administrators monitoring the Operator status. With this update, if the Operator encounters a momentary error, it no longer indicates `openshift-samples` as having `Degraded` status and retries to connect to the API server. Momentary shifts to `Degraded` status should no longer occur. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1993840[*BZ#1993840*])
+* Before this update, if the Cluster Samples Operator encountered an `APIServerConflictError` error, it reported `sample-operator` as having `Degraded status` until it recovered. Momentary errors of this type were not unusual during upgrades but caused undue concern for administrators monitoring the Operator status. With this update, if the Operator encounters a momentary error, it no longer indicates `openshift-samples` as having `Degraded status` and tries again to connect to the API server. Momentary shifts to `Degraded status` no longer occur. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1993840[BZ#1993840])
+
+* Before this update, various allowed and blocked registry configuration options in the cluster image configuration might prevent the Cluster Samples Operator from creating image streams. As a result, the samples operator might mark itself as degraded, which impacted the general {product-title} install and upgrade status.
++
+In various circumstances, the management state of the Cluster Samples Operator can transition to `Removed`. With this update, these circumstances include when the image controller configuration parameters prevent the creation of image streams by using either the default image registry or the image registry specified by the `samplesRegistry` setting. The Operator status also indicates that the cluster image configuration is preventing the creation of the sample image streams. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002368[BZ#2002368])
```

Comment:
```suggestion
In various circumstances, the management state of the Cluster Samples Operator can transition to `Removed`. With this update, these circumstances now include when the image controller configuration parameters prevent the creation of image streams by using either the default image registry or the image registry specified by the `samplesRegistry` setting. The Operator status now also indicates that the cluster image configuration is preventing the creation of the sample image streams. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002368[BZ#2002368])
```
This description needs better "then vs now" contrast. As phrased, I'm not entirely clear what has changed with the update.

Reply:
Updated. Thanks.